docs/UserGuide.md

```diff
@@ -283,7 +280,7 @@ Example:
 ### 2.18. Complete or Make Uncomplete a Task : `mark`
 
 Marks a task to be complete or incomplete.
-Format: `mark [complete/uncomplete] INDEX`
+Format: `mark [complete/incomplete] INDEX`
 
 > * Using complete as a parameter will mark the selected task as complete.
 > * Using incomplete as a parameter will mark the selected task as incomplete.
```

Comment:
Using neither, should it be toggle instead? Or mark as complete (if so, this kinda defeats the purpose of mark complete 1)